Packet++/header/ProtocolType.h

```diff
@@ -267,6 +267,11 @@ namespace pcpp
 	 */
 	const ProtocolType FTP = 0x10000000000;
 
+	/**
+	 * SOME/IP Base protocol
+	 */
+	const ProtocolType SomeIP = 0x20000000000;

```

Comment:
Please merge your branch from `dev` - the value `0x20000000000` is already taken by `ICMPv6`, it should be `0x40000000000`